namui/src/namui/render/rect.rs

```diff
@@ -47,37 +47,49 @@ pub fn rect(
     }: RectParam,
 ) -> RenderingTree {
     let mut rendering_tree: Vec<RenderingTree> = vec![];
-    let rect: namui::XywhRect<f32> = match stroke {
+    let (x, y, rect) = match stroke {
```

Comment:
I extract x and y from rect and used `translate()` because of PathBuilder cache.